tree-sitter-markdown/test/corpus/spec.txt

```diff
@@ -1988,8 +1988,10 @@ Example 117 - https://github.github.com/gfm/#example-117
     (fenced_code_block
       (fenced_code_block_delimiter)
       (block_continuation)
-      (code_fence_content
-        (block_continuation))
+      (ERROR
```

Comment:
Is this `ERROR` expected?

Reply:
I have no idea what this grammars expect.
GitHub seems to render the way you've mentioned indeed: https://gist.github.com/SomeoneToIgnore/6c87cec69e6aa29fac76cb8ac7230912

Do you have any ideas how to fix it properly?

Reply:
No, sorry. I'd suspect the scanner needs to lookahead further to the end of the line, and then check if there are any non-whitespace characters.

Reply:
What complicates things (besides my diminishing knowledge of the area) is the fact that pure `split_parser` branch fails for me in many tests:
```
  extension_wikilink:
    ✗ Basic Wiki-link parsing.
    ✗ Wiki-link to a file
    ✗ Wiki-link to a heading in a note
    ✗ Wiki-link with title
    ✓ Wiki-link opener with no closer
    ✗ Wiki-link version of Example 556
  failing:
     Example 355 - https://github.github.com/gfm/#example-355
     Example 363 - https://github.github.com/gfm/#example-363
     Example 389 - https://github.github.com/gfm/#example-389
     Example 394 - https://github.github.com/gfm/#example-394
     Example 400 - https://github.github.com/gfm/#example-400
     Example 401 - https://github.github.com/gfm/#example-401
     Example 407 - https://github.github.com/gfm/#example-407
     Example 406 - https://github.github.com/gfm/#example-406
     Example 411 - https://github.github.com/gfm/#example-411
     Example 420 - https://github.github.com/gfm/#example-420
     Example 421 - https://github.github.com/gfm/#example-421
     Example 424 - https://github.github.com/gfm/#example-424
     Example 438 - https://github.github.com/gfm/#example-438
     Example 524 - https://github.github.com/gfm/#example-524
     Example 538 - https://github.github.com/gfm/#example-538
     Example 560 - https://github.github.com/gfm/#example-560
     Example 588 - https://github.github.com/gfm/#example-588
     Example 635 - https://github.github.com/gfm/#example-635
```

which does not simplify things.
So I'd love some guidance.

Reply:
Sorry that I did not answer in time. As clason mentioned, the parser should not output an error here. The problem is, that you do not "consume" the trailing whitespaces. There needs to be another while loop like the one above, but with whitespace instead of `delimiter`. (If you still want to work on this.)

I'm not sure why the tests fail for you. Try using the instructions [here](https://github.com/tree-sitter-grammars/tree-sitter-markdown/blob/split_parser/CONTRIBUTING.md#tests) if you didn't already.

Reply:
Sorry, I didn't see there was another PR that fixed this problem already.